<?xml version="1.0" encoding="utf-8"?><testsuite errors="0" failures="1" name="pytest" skips="0" tests="16" time="0.083"><testcase classname="test_module01" file="test_module01.py" line="6" name="test_case01" time="0.000997781753540039"></testcase><testcase classname="test_module02.TestClass01" file="test_module02.py" line="12" name="test_case01" time="0.00034427642822265625"></testcase><testcase classname="test_module02.TestClass01" file="test_module02.py" line="15" name="test_case02" time="0.0004336833953857422"></testcase><testcase classname="test_module03" file="test_module03.py" line="26" name="test_case01" time="0.0006451606750488281"></testcase><testcase classname="test_module03" file="test_module03.py" line="30" name="test_case02" time="0.0003948211669921875"></testcase><testcase classname="test_module03.TestClass02" file="test_module03.py" line="50" name="test_case03" time="0.0003829002380371094"></testcase><testcase classname="test_module03.TestClass02" file="test_module03.py" line="53" name="test_case04" time="0.0003838539123535156"></testcase><testcase classname="test_module04" file="test_module04.py" line="8" name="test_case01" time="0.0005366802215576172"></testcase><testcase classname="test_module05" file="test_module05.py" line="10" name="test_case01" time="0.0005097389221191406"></testcase><testcase classname="test_module06.TestClass03" file="test_module06.py" line="14" name="test_case01" time="0.0006697177886962891"></testcase><testcase classname="test_module06.TestClass03" file="test_module06.py" line="17" name="test_case02" time="0.0005228519439697266"></testcase><testcase classname="test_module07" file="test_module07.py" line="16" name="test_case01" time="0.0024862289428710938"></testcase><testcase classname="test_module07" file="test_module07.py" line="21" name="test_case02" time="0.0013930797576904297"></testcase><testcase classname="test_module08" file="test_module08.py" line="22" name="test_case01" time="0.0008461475372314453"></testcase><testcase classname="test_module09" file="test_module09.py" line="3" name="test_case01" time="0.00040221214294433594"></testcase><testcase classname="test_module09" file="test_module09.py" line="8" name="test_case02" time="0.0004391670227050781"><failure message="Failed: DID NOT RAISE &lt;class &apos;Exception&apos;&gt;">def test_case02():
        with pytest.raises(Exception):
&gt;           x = 1 / 1       # Nope!
E           Failed: DID NOT RAISE &lt;class &apos;Exception&apos;&gt;

test_module09.py:11: Failed</failure></testcase></testsuite>